tdesign/src/themes/muiTheme.ts

```diff
@@ -188,44 +190,53 @@ export const muiTheme = ({
         "-apple-system, BlinkMacSystemFont, avenir next, avenir, segoe ui, helvetica neue, helvetica, Ubuntu, roboto, noto, arial, sans-serif;",
       // Default to sans-serif, b/c of all the text styles 2 are serif and the rest sans. List from systemfontstack.com -> sans-serif
       title1: {
+        //H1 Title - SF SB 24 pt
```

Comment:
I have trouble deciphering what this means. SF SB 24pt? Can you explain :smile: ? Possibly by adding a comment in the source code itself, so it says for longer than the lifetime of this PR :slightly_smiling_face:

Reply:
This is the nomenclature used by Flambee to describe the font styles they designed. I just found it incredibly slow and unintuitive to have the font style in code named `title1` or `title2` but in the designs referred to as `H1 Title` and `H2 Subtitle` respectively. 

I'll add this link to the comments: https://www.figma.com/file/jaOp7ncR6Zk90BeOQgrIB0/--Splitgraph-%2F-Component-and-Styles-Library

Reply:
Makes sense :+1: